recipes/jwt-cpp/all/test_package/test_package.cpp

```diff
@@ -1,4 +1,5 @@
 #include <jwt-cpp/jwt.h>
+#include <jwt-cpp/traits/kazuho-picojson/defaults.h>
```

Comment:
this only exists in 0.6.0 + 

You'll need to drop older versions 🤔

Reply:
I added some additional definitions to the test package and placed picojson traits files to the test package folder to be able to compile older jwt-cpp versions.
Tested building on all versions listed in conandata.yml

Reply:
How am I supposed to use this package now? Why do you want me to copy-paste these files as well, instead of just changing _supports_generic_json to version >= 6.0. I see no reason why we want to define JWT_DISABLE_PICOJSON in the recipe.

Reply:
This is not an example, the test package is more complicated because it's supporting many versions

Reply:
If you use version  0.6.0 you can include any of the support JSON libraries and that file will already be present

Reply:
I updated 0.5.0 to new revision and it became not usable because of hard coded define :(
I had to upgrade to 0.6.0 and slightly adjust my code, now it is working. But I still think setting this define without option flag was a bad decision. It turns off part of the functionality of the library which I was using in the previous revision.

Reply:
https://github.com/Thalhammer/jwt-cpp/releases did add support for generic json at 0.5.0 but I agree  this could be improved https://github.com/conan-io/conan-center-index/pull/12735#pullrequestreview-1107550592

Feel free to open a PR to improve the recipe :)